_im-plugin/index-rollups/index.md

```diff
@@ -454,3 +454,265 @@ POST example_rollup/_search
+            } 
+          } 
+        ], 
+        "transitions": [ 
+          { 
+            "state_name": "rp" 
+          } 
+        ] 
+      }, 
+      { 
+        "name": "rp", 
+        "actions": [
+          { 
+            "rollup": { 
+              "ism_rollup": { 
+                "target_index": "rollup_ndx-{{ctx.source_index}}", 
+                "description": "Example rollup job", 
+                "page_size": 200, 
+                "dimensions": [ 
+                  { 
+                    "date_histogram": { 
+                      "source_field": "ts", 
+                      "fixed_interval": "60m", 
+                      "timezone": "America/Los_Angeles" 
+                    } 
+                  }, 
+                  { 
+                    "terms": { 
+                      "source_field": "message.keyword" 
+                    } 
+                  } 
+                ], 
+                "metrics": [ 
+                  { 
+                    "source_field": "msg_size", 
+                    "metrics": [ 
+                      { 
+                        "sum": {} 
+                      } 
+                    ]
+                  } 
+                ]
+              } 
+            } 
+          } 
+        ], 
+        "transitions": [] 
+      } 
+    ], 
+    "ism_template": { 
+      "index_patterns": ["log*"], 
+      "priority": 100 
+    } 
+  } 
+}
+```
+
+**Step 2:** Create an index named `log-000001` and set up an alias `log` for it.
+
+```json
+PUT log-000001
+{
+  "aliases": {
+    "log": {
+      "is_write_index": true
+    }
+  }
+}
+```
+
+**Step 3:** Add an index template for ISM to manage the rolling over of the indices aliased by `log`.
+
+```json
+PUT _index_template/ism_rollover
+{
+  "index_patterns": ["log*"],
+  "template": {
+   "settings": {
+    "plugins.index_state_management.rollover_alias": "log"
+   }
+ }
+}
+```
+
+**Step 4:** Index four documents into the index created above: two of the documents have the message "Success", and two have the message "Error".
+
+```json
+POST log/_doc?refresh=true 
+{ 
+  "ts" : "2022-08-26T09:28:48-04:00", 
+  "message": "Success", 
+  "msg_size": 10 
+}
+```
+
+```json
+POST log/_doc?refresh=true 
+{ 
+  "ts" : "2022-08-26T10:06:25-04:00", 
+  "message": "Error", 
+  "msg_size": 20 
+}
+```
+
+```json
+POST log/_doc?refresh=true 
+{ 
+  "ts" : "2022-08-26T10:23:54-04:00", 
+  "message": "Error", 
+  "msg_size": 30 
+}
+```
+
+```json
+POST log/_doc?refresh=true 
+{ 
+  "ts" : "2022-08-26T10:53:41-04:00", 
+  "message": "Success", 
+  "msg_size": 40 
+}
+```
+
+Once you index the first document, the rollover action is executed. This action creates the index `log-000002` with `rollover_policy` attached to it. Then, the rollup action is executed, which creates the rollup index `rollup_ndx-log-000001`.
+
+To monitor the status of rollover and rollup indices' creation, you can use the ISM explain API: `GET _plugins/_ism/explain`
```

Comment:
Can we change "indices'" to "index" ("To monitor the status of rollover and rollup index creation...")?